view/stache/doc/helpers/is.md

```diff
@@ -0,0 +1,27 @@
+@function can.stache.helpers.is {{#is expr1 expr2 expr3}}
+@parent can.stache.htags 12
+
+@signature `{{#is expr1 expr2}}BLOCK{{/is}}`
+
+Renders the `BLOCK` template within the current template.
+
+@param {can.stache.key} [args] A key that references a value within the current or parent
```

Comment:
We don't unfortunately have a can.stache.expression that you can link to.  However, I can fill that in.  Can you change this to:

```
@param {can.stache.expression} [expr...] An expression or key that references a value within the current or parent
```